source/process/asset-guidelines.rst

```diff
@@ -0,0 +1,217 @@
+
+The following guidelines are intended to help queue and prepare marketing assets for community projects by the Mattermost core committers and the Mattermost community.
+
+.. contents::
+    :backlinks: top
+
+Queuing Assets for New Community Projects
+------------------------------------------
+
+Follow these steps to queue an asset for a new community project.
+
+Integrations and Installers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you find a new integration or installer in the `Mattermost Facebook page <https://www.facebook.com/MattermostHQ/?fref=ts>`_, `Mattermost Twitter page <https://twitter.com/mattermosthq>`_, `GitHub <https://github.com/search?utf8=%E2%9C%93&q=mattermost>`_ or elsewhere you feel Mattermost should promote:
+
+1. Post a request to the `Contributors team "Integrations and Apps" channel <https://pre-release.mattermost.com/core/channels/integrations>`_ with:
+
+  - mention to @jason
+  - link to suggested integration
+  - link to creator's GitHub profile, if applicable
+
+    Example:
+
+    .. code-block:: none
+
+      @jason Here is [Matterbridge](https://github.com/42wim/matterbridge) from [42wim](https://github.com/42wim), which is a sample bridge between Mattermost, IRC, XMPP, Gitter and Slack.
+      
+      Is this something Mattermost would like to promote?
+
+2. If the proposal is accepted, Ops reaches out to the integration creator by opening a Github issue with the following text:
+
+    .. code-block:: none
+
+      Title: 
+
+      Share your work with Mattermost community?
+
+      Body: 
+
+      Hi @{GitHub_username}, 
+
+      We came across your work and we'd love to help share it with the Mattermost community.
+
+      If this interests you, we wondered if you'd help complete a form to tell us more about your work so we can promote it? 
+      
+      Here's the form: https://spinpunch.wufoo.com/forms/mattermost-integrations-and-installers/
+      
+      We'd also like to welcome you to join our public integrations community channel on our nightly build server: https://pre-release.mattermost.com/core/channels/integrations
+
+      Thanks for being a valued member of the Mattermost Community!
+
+      - The Mattermost Team
+
+3. When a contributor submits an integration via the `Wufoo form <https://spinpunch.wufoo.com/forms/mattermost-integrations-and-installers/>`_, a Zapier webhook posts relevant information into the `Integrations and Apps channel <https://pre-release.mattermost.com/core/channels/integrations>`_.
+
+4. Ops replies to each new Wufoo post with a numbered project title and project status. Example:
+
+    .. code-block:: none
+
+      #### 64) Cloud Foundry Integration (work queued)
+
+Media Articles
+^^^^^^^^^^^^^^^
+
+When a relevant article on Mattermost is found in the news, add a new entry into `this Google document <https://docs.google.com/document/d/1kwCmn6JYeORdLV0noIk4iaxZx0iqR6OWUuzw5cZl6rA/edit>`_ and follow the process specified there.
```

Comment:
I made a couple of comments in the Google doc as it seems a few links are out of date?